docs/src/main/sphinx/admin/properties-writer-scaling.rst

```diff
@@ -33,3 +33,11 @@ another writer is eligible to be added. Each writer task may have multiple
 writers, controlled by ``task.writer-count``, thus this value is effectively
 divided by the number of writers per task. This can be specified on a
 per-query basis using the ``writer_min_size`` session property.
+
+``use-preferred-write-partitioning``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+* **Type:** ``boolean``
+* **Default value:** ``false``
+
+Enable preferred write partitioning. This option will force each partition
```

Comment:
Also add:
`As a result, for some connectors (e.g Hive) there will be only a single new file per partition instead of multiple ones. Partition writer assignments are distributed across worker nodes`.

Reply:
done